tensorflow/core/platform/s3/BUILD

```diff
@@ -102,6 +107,12 @@ tf_cc_test(
     tags = [
         "manual",
     ],
+    args = select({
+        "@org_tensorflow//tensorflow:windows": [
+            "--dynamic_mode=off",
+        ],
```

Comment:
This flag is needed to compile anything on windows, let's add it to .bazelrc instead

Reply:
It is ok ? Since adding to `.bazelrc` will cause everything to rebuild

Reply:
actually, this does not work and I have to add `--dynamic_mode=off` manually when build. So I will remove it in the next commit. Maybe the reason is we have to add to its dependencies as well

Reply:
I think it's because it is a Bazel flag, not a compiler flag. so it needs to be passed to bazel directly.